docs/useAzkaban.rst

```diff
@@ -245,14 +245,23 @@ executing, several options can be set.
 
 .. image:: figures/executeflowconcurrent.png
 
-Flow Parameters
+Runtime Properties
 ~~~~~~~~~~~~~~~
 
-Allows users to override flow parameters. The flow parameters override
-the global properties for a job, but not the properties of the job
-itself.
+Runtime Properties allow users to override properties temporarily.
 
-.. image:: figures/executeflowparameters.png
+Note: This feature was previously known as Flow Parameters.
+
+.. image:: figures/executeflowruntimeproperties.png
+
+The scope of each runtime property is defined by the Node selection. Rows with `ROOT` Node apply to all jobs in the flow. Properties can be scoped per sub-flow as well as individual jobs. Most specific node path takes precedence.
+
+Overrides are applied depending on the Azkaban server configuration property `executor.props.resolve.overrideExisting.enabled`:
```

Comment:
I can see the possibility for confusion. How about this: `executor.runtimeProps.override.eager`? The default would be `false`. Or did you have some name in mind that you'd like to be used?

Reply:
I think `executor.runtimeProps.override.eager` is perfect (with `azkaban.` prefix even more :) ) because it hints at changing the behavior of _overrides_ instead of disabling them and is ambiguous enough that users will have to read the documentation instead of making assumptions. Thanks!